src/__tests__/extensions/replay/sessionrecording.test.ts

```diff
@@ -629,6 +689,21 @@ describe('SessionRecording', () => {
             })
         })
 
+        it('stores masking config in persistence if set on the server', () => {
```

Comment:
we could add a test that proves that the received masking options are what are passed in to rrweb, since that after all is what the feature relies on